src/webob/acceptparse.py

```diff
@@ -1014,619 +822,97 @@ def acceptable_offers(self, offers):
 
     def best_match(self, offers, default_match=None):
         """
-        Return the best match from the sequence of media type `offers`.
+        Return the best match from the sequence of media type ``offers``.
 
-        .. warning::
+        This is a thin wrapper around :meth:`.acceptable_offers` that makes
+        usage more convenient for typical use-cases where you just want
+        to know the client's most preferred match.
 
-           This is currently maintained for backward compatibility, and will be
-           deprecated in the future.
+        :param offers:
+            (iterable)
 
-           :meth:`AcceptValidHeader.best_match` uses its own algorithm (one not
-           specified in :rfc:`RFC 7231 <7231>`) to determine what is a best
-           match. The algorithm has many issues, and does not conform to
-           :rfc:`RFC 7231 <7231>`.
+            | Each item in the iterable must be a ``str`` media type and
+              may contain params/extensions.
 
-        Each media type in `offers` is checked against each non-``q=0`` range
-        in the header. If the two are a match according to WebOb's old
-        criterion for a match, the quality value of the match is the qvalue of
-        the media range from the header multiplied by the server quality value
-        of the offer (if the server quality value is not supplied, it is 1).
+        :param default_match:
+            (optional, any type) the value to be returned if there is no match
 
-        The offer in the match with the highest quality value is the best
-        match. If there is more than one match with the highest qvalue, the
-        match where the media range has a lower number of '*'s is the best
-        match. If the two have the same number of '*'s, the one that shows up
-        first in `offers` is the best match.
+        :return:
+            (``str``, or the type of ``default_match``)
 
-        :param offers: (iterable)
+            | The offer that is the best match based on q-value. If there is no
+              match, the value of ``default_match`` is returned. Where two
+              offers match the same qvalue, they are returned in the same order
+              as their order in ``offers``.
```

Comment:
`.best_match` returns a single offer, so the "they are returned" seems a little confusing?

Also, inconsistent spelling of "qvalue"/"q-value"? (RFC spells it without a hyphen.)